react-native-libraries.json

```diff
@@ -5126,5 +5126,18 @@
     "expo": false,
     "windows": false,
     "macos": false
+  },
+  {
+    "githubUrl": "https://github.com/sospedra/reinput",
+    "npmPkg": "https://www.npmjs.com/package/reinput",
```

Comment:
Oh, I have just noticed that, sorry. You need to remove `npmPkg` or change it to the package name.

Reply:
np, done ^^